types/msg.go

```diff
@@ -31,6 +31,37 @@ type Response struct {
 	Events []Event `json:"events"`
```

Comment:
Why not use the `Events` type alias here?

Reply:
Probably historic reasons. It was only needed when adding the Go->Rust way with submessage responses. For the Rust->Go way it was not necessary. 

Since we have the same issue with `type EventAttributes []EventAttribute`, I would like to avoid changing it now. This whole nil vs. [] thing scares me.